app/helper/RTMediaModel.php

```diff
@@ -253,38 +283,39 @@ function get_user_albums( $author_id, $offset, $per_page, $order_by = 'media_id
 			$sql .= ' LIMIT ' . $offset . ',' . $per_page;
 		}
 
-		$results = $wpdb->get_results( $sql ); // @codingStandardsIgnoreStart
+		$results = $wpdb->get_results( $sql ); // @codingStandardsIgnoreEnd
```

Comment:
Use `// phpcs:disable` instead of `@codingStandardsIgnoreStart` and `// phpcs:enable` instead of `@codingStandardsIgnoreEnd`